lib/src/input_validation/pattern.dart

```diff
@@ -1,9 +1,8 @@
 // This class represents the possible Regex patterns used for input validation
 class Pattern {
-  static final validOperand = RegExp(r'^-?[0-9]+(.[0-9]+)?$|^[𝜋𝑒]$', unicode: true);
+  static final validOperand = RegExp(r'^(\d*\.?\d+)?(𝜋|𝑒|²|⁻¹)*$', unicode: true);
```

Comment:
the ² and ⁻¹ shouldn't be valid operands

Reply:
i agree